### ISSUE TITLE
Crash when signing out or deleting account

### ISSUE THREAD

**Issue by RohanNagar** [SanctionCo/pilot-ios#61]
Due to the removal of storyboards, the app crashes when trying to execute this line in `ProfileViewController`:

```swift
let storyboard = UIStoryboard.init(name: "LoginViewController", bundle: nil)
```

Needs to be fixed as we update the settings views.

**Comment by RohanNagar** [comment]
@NickEckert was this fixed in #77?

**Comment by NickEckert** [comment]
Yes, Fixed in #77
